shared/ui/components/form/Form.js

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { Button } from '@aragon/ui'
+
+const Form = ({
+  children,
+  className,
+  onSubmit,
+  submitText,
+  disabled,
+  errors,
+}) => {
+  return (
+    <form
+      className={className}
+      onSubmit={(e) => {
+        e.preventDefault()
+        onSubmit()
+      }}
+    >
+      <VerticalSpace />
+      {children}
+      <Button
+        type="submit"
+        style={{ userSelect: 'none', marginTop: '24px' }}
```

Comment:
ugh, I don't know why we add `user-select: none` all over the place; I often get frustrated by UIs that don't let me select text

Reply:
Well @chadoh why would you want to select a Button label? Even I agree with you about webs abusing `user-select: none`, for these cases of elements UI, I don't think is that bad, users normally wants to select the content but not the interface elements text. I also don't normally see `user-select: none` all over the place.

We should difference between content and UI elements, the latter rarely provide any interesting information for the user more than the needed to interact with the element, as opposed to the content type, that it was normally the user would want to select/copy 

Said this, there is no need to add this style as the [`@aragon/ui` `Button`](https://ui.aragon.org/button/) already [has it ](https://github.com/aragon/aragon-ui/blob/38bda15486985ce9fa0959ddb09ca7b076982e50/src/components/Button/Button.js#L250) applied, even if we need to use the style for whatever reason, the correct way is to use the `unselectable` helper from `@aragon/ui` @PeterMPhillips

Reply:
haha, I was trying to select the "comment" button text and sent my comment before I meant to! The "comment" button here does not have `user-select: none`, but I still can't select its text.

Anyhow, y'all are seeing my annoyance at having run into problems with various apps, Aragon apps and others, which have prevented me from selecting text. I just can't figure out why this is a trend right now. I guess maybe `user-select` is still new enough that people are still over-using it.

Anyhow, yeah, aragonUI already adds this annoying rule, so there's no need for us to add it unless we want to override it back to `user-select: auto`.

Reply:
> To clarify, I don't think aragonUI should set `user-select: none` almost ever, maybe not at all
> 
> I don't think we should ask "why would a user ever want to select such-and-such text?" Instead, I think we should ask "why should we prevent them from selecting it?" Who knows what users will do or why they would want to do it! Trying to guess at all possible motives for any given action they might take seems futile. Maybe they want to select it so they can _ask us a question about it_, and that is valid and we are preventing it. But even if I couldn't come up with such a reason, that doesn't prove that no valid reason exists.
> 
> Clicking buttons already works properly. I don't see any value in limiting the possible uses of the text in a button so that the occassional user who tries to select the text will get frustrated.

The same thing could apply to the question: "why don't we just publish source code and avoid creating UIs at all? that way users would be able to freely access everything in the app... Yes, it sounds ridiculous, is not about letting the user select one thing or another, a enough experimented user (a developer, for example) will be able to select the text if he wants, it is more a cosmetic, conventional thing, try to select the text from buttons in github? oh, you cannot do that, also, can you do that on your desktop apps, or phone? Definitely not. Is also not needed, there is not problem with that, because nobody needs to select the text in a button, because there should not be content on it, is just part of the UI, you can have whatever opinion about it, but is something conventional since the invention of GUIs. The rule exists for a reason.

Reply:
> haha, I was trying to select the "comment" button text and sent my comment before I meant to! The "comment" button here does not have `user-select: none`, but I still can't select its text.

Of course it has the rule, try to go to the "Computed" tab in your stylesheet inspector 😉

Reply:
I agree with Otto that I don't think it's important for button text to be copied